src/DependencyInjection/SonataUserExtension.php

```diff
@@ -66,7 +66,10 @@ public function load(array $configs, ContainerBuilder $container): void
 
         $loader->load('form.xml');
 
-        if (class_exists('Google\Authenticator\GoogleAuthenticator')) {
+        // The 'Google\Authenticator' namespace is deprecated in sonata-project/GoogleAuthenticator
+        // since version 2.1 and will be removed in 3.0.'
```

Comment:
I meant throw deprecation from php
```
@trigger_error(
    'The 'Google\Authenticator' namespace is deprecated in sonata-project/GoogleAuthenticator since version 2.1 and will be removed in 3.0.',
    E_USER_DEPRECATED
);

Reply:
if i do that the build can't pass and it's normal IMO. I already did this. I did it again so you can see

Reply:
thx for the link ;) my bad i'm pretty new to deprecate things. i tried so tell me if it's ok